### PR TITLE
ed: Sync with NTFS specs, some renaming and text_color in lines

### PR DESCRIPTION
**Pull request by azime**
Synchronization with NTFS format:

> Replace "property_id" by "equipment_id" in "transfers.txt" and "stops.txt" files
> Add "text_color" in "lines.txt" file
> Update model database
> tests added

### CODE REVIEW

**Comment by azime** [comment]
corrections

File: source/ed/tests/fusioparser_test.cpp [hunk 1]
```diff
@@ -48,6 +48,7 @@ struct logger_initialized {
 BOOST_GLOBAL_FIXTURE( logger_initialized )
 
 const std::string ntfs_path = std::string(navitia::config::fixtures_dir) + "/ed/ntfs";
+const std::string sync_ntfs_path = std::string(navitia::config::fixtures_dir) + "/ed/sync_ntfs";
```
Comment (antoine-de) [inline source/ed/tests/fusioparser_test.cpp]: what is this sync_ntfs ?

File: source/ed/tests/gtfsparser_test.cpp [hunk 2]
```diff
@@ -362,7 +362,7 @@ static void check_gtfs_google_example(const ed::Data& data, const ed::connectors
     BOOST_REQUIRE_EQUAL(data.stop_point_connections.size(), 0);
 
     BOOST_REQUIRE_EQUAL(data.lines.size(), 5);
-    BOOST_CHECK_EQUAL(data.lines[0]->uri, "AB");
+    BOOST_CHECK_EQUAL(data.lines[0]->uri, "AB");
```
Comment (antoine-de) [inline source/ed/tests/gtfsparser_test.cpp]: wrong space

File: fixtures/ed/gtfs_google_example/routes.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 route_id,agency_id,route_short_name,route_long_name,route_desc,route_type,route_url,route_color,route_text_color
-AB,DTA,10,Airport - Bullfrog,,3,,,
+AB,DTA,10,Airport - Bullfrog,,3,,,FFD700
```
Comment (antoine-de) [inline fixtures/ed/gtfs_google_example/routes.txt]: hum this directory is the raw google GTFS example, I think it's better if we do not change it.

Maybe you can change another GTFS fixture ?

File: source/type/type.h [hunk 4]
```diff
@@ -622,7 +623,7 @@ struct Line : public Header, Nameable, HasMessages {
 
     template<class Archive> void serialize(Archive & ar, const unsigned int ) {
         ar & idx & name & uri & code & forward_name & backward_name
-                & additional_data & color & sort & commercial_mode
+                & additional_data & color & text_color& sort & commercial_mode
```
Comment (antoine-de) [inline source/type/type.h]: spacing

File: source/ed/connectors/gtfs_parser.cpp [hunk 5]
```diff
@@ -563,6 +563,7 @@ void RouteGtfsHandler::init(Data&) {
             long_name_c = csv.get_pos_col("route_long_name"), type_c = csv.get_pos_col("route_type"),
             desc_c = csv.get_pos_col("route_desc"),
             color_c = csv.get_pos_col("route_color"), agency_c = csv.get_pos_col("agency_id");
+    text_color_c = csv.get_pos_col("route_text_color");
```
Comment (antoine-de) [inline source/ed/connectors/gtfs_parser.cpp]: I think there is an indent problem

File: source/type/type.h (hunk 4 above)
Comment (azime) [inline source/type/type.h]: OK

File: source/ed/tests/gtfsparser_test.cpp (hunk 2 above)
Comment (azime) [inline source/ed/tests/gtfsparser_test.cpp]: OK

File: source/ed/tests/fusioparser_test.cpp (hunk 1 above)
Comment (azime) [inline source/ed/tests/fusioparser_test.cpp]: I need to change the names of columns and some files

File: source/ed/connectors/gtfs_parser.cpp (hunk 5 above)
Comment (azime) [inline source/ed/connectors/gtfs_parser.cpp]: OK

File: fixtures/ed/gtfs_google_example/routes.txt (hunk 3 above)
Comment (azime) [inline fixtures/ed/gtfs_google_example/routes.txt]: https://developers.google.com/transit/gtfs/reference#routes_fields
route_text_color is gtfs data

File: source/ed/tests/fusioparser_test.cpp (hunk 1 above)
Comment (antoine-de) [inline source/ed/tests/fusioparser_test.cpp]: It's just that the name of the dir is not very clear.

It's a fixture with the new version of ntfs ? can we call it /ed/ntfs_v{N} ?

File: fixtures/ed/gtfs_google_example/routes.txt (hunk 3 above)
Comment (antoine-de) [inline fixtures/ed/gtfs_google_example/routes.txt]: It's just that this fixture is the example GTFS  google provides. I think it's thus better for this fixture not to be changed

File: source/ed/tests/fusioparser_test.cpp (hunk 1 above)
Comment (azime) [inline source/ed/tests/fusioparser_test.cpp]: OK

File: fixtures/ed/gtfs_google_example/routes.txt (hunk 3 above)
Comment (azime) [inline fixtures/ed/gtfs_google_example/routes.txt]: OK

**Comment by azime** [comment]
corrections

File: fixtures/ed/gtfs_google_example/routes.txt [hunk 6]
```diff
@@ -3,4 +3,4 @@ AB,DTA,10,Airport - Bullfrog,,3,,,
 BFC,DTA,20,Bullfrog - Furnace Creek Resort,,3,,,
 STBA,DTA,30,Stagecoach - Airport Shuttle,,3,,,
 CITY,DTA,40,City,,3,,,
-AAMV,DTA,50,Airport - Amargosa Valley,,3,,,
```
Comment (antoine-de) [inline fixtures/ed/gtfs_google_example/routes.txt:6]: What did you change on this line ?

**Comment by azime** [comment]
corrections

**Pull request merged**